README.md

```diff
@@ -8,10 +8,14 @@ Compatibility
 The following Grove compatible devices are supported in Python:
 
 * Dexter Industries:
-  * Distance Sensor
-  * Inertial Measurement Unit Sensor
-  * Light Color Sensor
-  * Temperature Humidity Pressure Sensor
+  * **[Distance Sensor](https://www.dexterindustries.com/shop/distance-sensor/)** - Distance Sensor for the GoPiGo, GrovePi, and BrickPi.  The distance sensor can be mounted to the GoPiGo Raspberry Pi robot with or without the servo package to enable rotation.  The sensor detects distances from obstacles and objects, giving your robot the ability to navigate.
+
+  * **[Inertial Measurement Unit Sensor](https://www.dexterindustries.com/shop/imu-sensor/)** - The IMU Sensor attaches to the GrovePi, GoPiGo and BrickPi to detect motion, orientation, and position of your robot. It has a compass, accelerometer, and gyroscope and allows you to build a BalanceBot.
+
+  * **[Light Color Sensor](https://www.dexterindustries.com/shop/light-color-sensor/)** - The Light & Color Sensor attaches to the GrovePi, GoPiGo and BrickPi to measure light levels and detect different colors. It can be used to build projects like a rubiks cube solver, weather station, or plant monitoring station.
+
+  * **[Temperature Humidity Pressure Sensor](https://www.dexterindustries.com/shop/temperature-humidity-pressure-sensor/) **- The Temperature Humidity and Pressure Sensor attaches to the GrovePi, GoPiGo and BrickPi to measure environmental conditions. It can be used to build projects like a classroom weather station or plant monitoring station.
```

Comment:
There's a space between the last ** and the actual link which makes the README look this way:
![format](https://user-images.githubusercontent.com/26958764/34829382-395c59ce-f6e9-11e7-9795-ffe8adf4b149.PNG)